junit5-extension/src/test/java/com/example/StubAnnotation.java

```diff
@@ -0,0 +1,11 @@
+package com.example;
```

Comment:
Debatable packaging choice, but it needs to be something outside of `io.kroxylicious`

Reply:
I agree that adding the annotation is the right thing to do.